common/src/main/java/com/microsoft/identity/common/adal/internal/AuthenticationConstants.java

```diff
@@ -148,6 +151,7 @@ private AuthenticationConstants() {
          * Waiting for broker package to be installed, and resume request in broker.
          */
         public static final int BROKER_REQUEST_RESUME = 2006;
+        public static final int BROWSER_CODE_DEVICE_REGISTER = 2006;
```

Comment:
Is this constant value dictated by a spec? It matches the same code as `BROKER_REQUEST_RESUME`

Reply:
this is a constant used in ad-accounts, it also has the value of '2006', to not break the build of ad-accounts, it is added here. Let's check if we could use the same variable name after Unity is deprecated.